### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 20

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#196] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152253899-ea5c4612-2061-4b82-9a93-aa302552ea31.png)

**Comment by neoSnakex34** [comment]
![image](https://user-images.githubusercontent.com/97003970/152637900-503f945c-f2aa-416c-aa9f-5bfb4ee0804f.png)

**Comment by Exyss** [comment]
Soluzioni analoghe a @neoSnakex34
